panel/config.py

```diff
@@ -79,6 +79,12 @@ class _config(param.Parameterized):
         embedded. Useful when only partial updates are made in an
         app, e.g. when working with HoloViews.""")
 
+    session_history = param.Integer(default=0, bounds=(-1, None), doc="""
+        If set to a non-zero value this determines the maximum length
```

Comment:
```suggestion
        If set to a non-negative value this determines the maximum length
```

(previously it was unclear what happens at zero, but I'm pretty sure zero just acts like positive numbers do, so I've reworded it to say so).